pwa/src/components/subscribers/subscriberForm.tsx

```diff
@@ -196,138 +218,85 @@ export const SubscriberForm: React.FC<SubscriberFormProps> = ({ subscriberId, en
+                        <SelectSingle
                           options={getEndpointsSelectQuery.data ?? []}
-                          data={subscriber?.endpoint?.name}
-                          name={"endpoint"}
-                          id={"endpointInput"}
-                          nameOverride={"Endpoint"}
-                          value={"admin/endpoints/"}
+                          name="endpoint"
+                          label="Endpoint"
+                          {...{ control, errors }}
                         />
                       </div>
                     </div>
                     <div className="row form-row">
                       <div className="col-12 col-sm-6 ">
-                        <div className="form-check">
-                          <Checkbox
-                            type={"checkbox"}
-                            id={"asynchronousInput"}
-                            nameLabel={"Asynchronous"}
-                            nameAttribute={"asynchronous"}
-                            data={subscriber?.asynchronous}
-                            defaultValue={"true"}
-                          />
-                        </div>
+                        <InputCheckbox name="asynchronous" label="Asynchronous" {...{ register, errors }} />
                       </div>
                       <div className="col-12 col-sm-6 ">
-                        <div className="form-check">
-                          <Checkbox
-                            type={"checkbox"}
-                            id={"blockingInput"}
-                            nameLabel={"Blocking"}
-                            nameAttribute={"blocking"}
-                            data={subscriber?.blocking}
-                          />
-                        </div>
+                        <InputCheckbox name="blocking" label="Blocking" {...{ register, errors }} />
                       </div>
                     </div>
 
                     <Accordion
                       id="handlerAccordion"
                       items={[
                         {
-                          title: "Translations In",
+                          title: "Translations in",
                           id: "translationsInAccordion",
                           render: function () {
                             return tableNames ? (
-                              <MultiSelect
-                                id="translationsIn"
-                                label="Translations In"
-                                data={subscriber?.translationsIn}
+                              <SelectMultiple
+                                name="translationsIn"
+                                label="Translations in"
                                 options={tableNames}
+                                {...{ control, errors }}
                               />
                             ) : (
                               <>
```

Comment:
This feedback goes for a couple of components within the accordions.

This accordion item is now set as followed:

```
{
  title: "Translations in",
  id: "translationsInAccordion",
  render: function () {
  return tableNames ? (    <== explicit return statement
    <SelectMultiple
      name="translationsIn"
      label="Translations in"
      options={tableNames}
      {...{ control, errors }}
    />
  ) : (
    <>    <== redundant wrapper
      <Spinner />
    </>
  );
},
```

I think we should update the following two aspects of this implementation (since we're refactoring the form anyway, we might as well):

1. We're using an explicit `return` statement in the `render` prop of this accordion item.  A arrow function returns its value by default if you **not not** include the curly brackets. So:

```
() => { // this does not return }
() => this does return

// and what we're doing..
() => { return // this does also return but has redundant code  }
``` 

2. If a value does not exist we want to return a `Spinner` (which is fine). However, we're wrapping this in another element, like so:

```
// current implementation
<>
  <Spinner />
< />

// a better implementation
<Spinner />
```

Reply:
Edit: for clarity, here's the final `render` prop that I think should be the implementation:


```
render: () =>                 <== no curly brackets and no return statement
  tableNames ? (
    <SelectMultiple
      name="translationsIn"
      label="Translations in"
      options={tableNames}
      {...{ control, errors }}
    />
  ) : (
    <Spinner />               <== no redundant wrapper
  ),
```